server/controllers/client.ts

```diff
@@ -6,8 +6,10 @@ import { asyncMiddleware } from '../middlewares'
 import { buildFileLocale, getCompleteLocale, is18nLocale, LOCALE_FILES } from '../../shared/models/i18n/i18n'
 import { ClientHtml } from '../lib/client-html'
 import { logger } from '../helpers/logger'
+import { advertiseDoNotTrack } from '../middlewares/dnt'
 
 const clientsRouter = express.Router()
+clientsRouter.use(advertiseDoNotTrack)
```

Comment:
You already added this middleware in `server.ts`

Reply:
It was not showing until I added it specifically there...

Reply:
We need to find why, because it's not normal